api/v1alpha1/integrationtestscenario_types.go

```diff
@@ -53,9 +54,9 @@ type PipelineParameter struct {
 
 // TestEnvironment contains the name and values of a Test environment
 type TestEnvironment struct {
-	Name   string   `json:"name"`
-	Type   string   `json:"type"`
-	Params []string `json:"params"`
+	Name          string                                          `json:"name"`
+	Type          string                                          `json:"type"`
```

Comment:
We could also set the `Type` to be [EnvironmentType](https://github.com/redhat-appstudio/application-api/blob/932f11859a29e8e413e9cd270e9d2cd3fa0a88ca/api/v1alpha1/environment_types.go#L54) instead of `string` to help with future compatibility.

Reply:
Updated, thanks!

Reply:
Thanks for the change!